test/integration/component_guide_test.rb

```diff
@@ -0,0 +1,15 @@
+require 'integration_test_helper'
```

Comment:
Should we have a specific test for the new component?

Reply:
Tests added in 35ef5ef